test/torchaudio_unittest/prototype/emformer_hubert_test.py

```diff
@@ -0,0 +1,99 @@
+import torch
+from parameterized import parameterized
+from torchaudio.prototype.models import emformer_hubert_base
+from torchaudio_unittest.common_utils import skipIfNoCuda, torch_script, TorchaudioTestCase
+
+
+class TestEmformerHuBERT(TorchaudioTestCase):
```

Comment:
do you think we could just reuse this tests file and use parameterized to include both conformerw2v2 and emformerhubert? https://github.com/pytorch/audio/blob/main/test/torchaudio_unittest/prototype/conformer_wav2vec2_test.py

Reply:
i think so, will update the existing test then

Reply:
I renamed the test file to `ssl_model_test.py` to reflect the change, let me know if you have a better idea.